### ISSUE TITLE
SCIRun crashes when node spheres are assigned a color

### ISSUE THREAD

**Issue by cbrightsci** [SCIInstitute/SCIRun#793]

**Comment by dcwhite** [comment]
@cbrightsci Is this the same bug? I got a crash when selecting nodes as spheres, message was:
```
Error compiling shader program with index 1:
0(78) : error C1035: assignment of incompatible types
0(78) : error C0000: syntax error, unexpected ')', expecting ',' or ';' at token
 ")"

Qt has caught an exception thrown from an event handler. Throwing
exceptions from an event handler is not supported in Qt. You must
reimplement QApplication::notify() and catch all exceptions there.

Unhandled exception: Failed to compile shader.
```

**Comment by cbrightsci** [comment]
Yes that is the same bug.

**Comment by dcwhite** [comment]
All right. It helps to have a bit of crash description, so we don't wind up with duplicate issues.

**Comment by brigb123** [comment]
fixed with 34b691e
